### ISSUE TITLE
Package is not installable via pip

### ISSUE THREAD

**Issue by fazpu** [mailersend/mailersend-python#10]
- `pip install mailersend` gives 

```
ERROR: Could not find a version that satisfies the requirement mailersend (from versions: none)
ERROR: No matching distribution found for mailersend
```

- python version: `3.7.8`
- pip version: `21.1.2`

**Comment by aorfanos** [comment]
I was able to install `mailersend` on a fresh environment both with python 2.7 and 3.8, not sure what could be wrong here.

**Comment by aorfanos** [comment]
Found it: package `mailersend` is available for versions `>=3.8, <4.0`. Reference in [PyPi](https://pypi.org/project/mailersend/).

**Comment by lowercase00** [comment]
Hm, curious that this is not available for 3.7, wonder what features prevent from supporting 3.7?
